odl/tomo/geometry/spect.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2014 - 2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SPECT geometry."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+import numpy as np
+
+from odl.tomo.geometry.geometry import AxisOrientedGeometry
+from odl.tomo.geometry.parallel import Parallel3dAxisGeometry
+
+__all__ = ('ParallelHoleCollimatorGeometry', )
+
+
+class ParallelHoleCollimatorGeometry(Parallel3dAxisGeometry):
+
+    """Geometry for SPECT Parallel hole collimator."""
+
+    def __init__(self, apart, dpart, axis=[0, 0, 1], det_rad=0.0,
```

Comment:
If no sensible default exists we should simply not take it as a parameter. Also for consistiency with the cone beam guys, this should come before `axis`.